css/components/sections/contact-me.css

```diff
@@ -0,0 +1,20 @@
+#contact-me > .container-fluid
+{
+    padding-left: 20%;
+    padding-right: 20%;
+}
+
+#send-btn {
```

Comment:
Rather than having an id of send-btn, you might instead add a modifier for send button with a name that describes how the 'style' is different from a normal button rather than your specific use of it. As a pattern this will lead to more reusable code.

Reply:
Great feedback! Will definitely work on it. Thank you.